region/brinstar/pink.json

```diff
@@ -997,23 +970,29 @@
                   "obstacles": [
                     {
                       "id": "A",
-                      "requires": [
-                        {"or":[
-                          "Bombs",
-                          {"and": [
-                            "PowerBomb",
-                            {"ammo": {
-                              "type": "PowerBomb",
-                              "count": 1
-                            }}
-                          ]}
-                        ]}
-                      ]
+                      "requires": [ "h_canBombThings" ]
                     }
                   ]
+                },
+                {
+                  "name": "Speedball",
+                  "notable": false,
+                  "requires": [ "Morph", "canMockball" ],
+                  "obstacles": [
+                    {
+                      "id": "A",
+                      "requires": [
+                        {"canComeInCharged": {
```

Comment:
Can't this can be done from Node 1 as well? Or would you need to jump before entering the room (which I don't believe is expressible yet)?